Jenkinsfile_CNP

```diff
@@ -6,7 +6,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "ccd"
-def camundaBranch = "master"
+def camundaBranch = "feat/CIV-6204"
```

Comment:
Presumably this needs to be reverted before merging. Gating the merge to prevent mistakes.